daemon/restapi/api_limits_test.go

```diff
@@ -9,39 +9,41 @@ import (
 	"fmt"
 	"testing"
 
+	"github.com/cilium/hive/cell"
+	"github.com/cilium/hive/hivetest"
 	"github.com/spf13/pflag"
 	"github.com/stretchr/testify/assert"
 	timeRate "golang.org/x/time/rate"
 
 	"github.com/cilium/cilium/pkg/hive"
-	"github.com/cilium/cilium/pkg/hive/cell"
 	"github.com/cilium/cilium/pkg/rate"
 )
 
 // TestRateLimiterConfigFlag checks that rateLimiterConfig is properly parsed from
 // command-line flag.
-func testRateLimiterConfig(t *testing.T, setConfig func(h *hive.Hive, limiter string, limits string)) {
+func testRateLimiterConfig(t *testing.T, burst int, setConfig func(h *hive.Hive, limiter string, limits string)) {
```

Comment:
The changes to the burst parameter aren't documented in the commit message or the PR. Are these changes related? Or accidentally lumped in?

Reply:
It's a broken test, due to shared global state in the pflag library. Will add a note to the commit message.